

if True:
    raise Exception('My error!')